src/components/header/AppHeaderMenu.vue

```diff
@@ -0,0 +1,64 @@
+<script>
+export default {
+  data() {
+    return {
+      items: [
+        //  Change the names to the correct names
+        {
+          name: 'Features',
+          url: '/features'
+        },
+        {
+          name: 'Prices',
+          url: '/prices'
+        },
+        {
+          name: 'Community',
+          url: '/community'
+        },
+        {
+          name: 'Support',
+          url: '/support'
+        }
+      ],
+      activeItem: -1
+
+    }
+
+  },
+
+  methods: {
+    clickHandler(idx) {
+      this.activeItem = idx
+    }
+  }
+
+}
+</script>
+<template>
+
+
+  <!-- Create an array and render the items -->
+  <!-- Create a function and bind it to the onclick. Add logic to change active menu item  -->
+  <li class="nav-item" v-for="(item, index) in items" 
+  :key="index" 
+  :class="{ active: index == activeItem }"
+  :v-on:click="clickHandler(index)">
```

Comment:
```suggestion
  v-on:click="clickHandler(index)">
```
no `:` before v-on
I would recommend using `@click` notation actually

```suggestion
  @click="clickHandler(index)">
```